blockchain/chain.go

```diff
@@ -594,7 +594,7 @@ func (b *BlockChain) connectBlock(node *blockNode, block *btcutil.Block,
 	blockSize := uint64(block.MsgBlock().SerializeSize())
 	blockWeight := uint64(GetBlockWeight(block))
 	state := newBestState(node, blockSize, blockWeight, numTxns,
-		curTotalTxns+numTxns, node.CalcPastMedianTime())
+		curTotalTxns+numTxns, CalcPastMedianTime(node))
```

Comment:
nit: should we fix the formatting of this multi-line function call if we're touching the code anyway? Here and below.

Reply:
wasn't sure what below is referencing?